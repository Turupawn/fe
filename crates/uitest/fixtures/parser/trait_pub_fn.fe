trait Fooable {
    pub fn foo(mut self)
    pub unsafe fn bar(self)
    fn x(self)
}
