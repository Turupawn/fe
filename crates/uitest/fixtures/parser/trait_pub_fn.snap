---
source: crates/uitest/tests/parser.rs
expression: diags
input_file: crates/uitest/fixtures/parser/trait_pub_fn.fe
---
error[1-0001]: `pub` modifier is not allowed in this block
  ┌─ trait_pub_fn.fe:2:5
  │
2 │     pub fn foo(mut self)
  │     ^^^ unexpected

error[1-0001]: `pub` modifier is not allowed in this block
  ┌─ trait_pub_fn.fe:3:5
  │
3 │     pub unsafe fn bar(self)
  │     ^^^ unexpected

error[1-0001]: `unsafe` modifier is not allowed in this block
  ┌─ trait_pub_fn.fe:3:9
  │
3 │     pub unsafe fn bar(self)
  │         ^^^^^^ unexpected


